tutorials/3d/spatial_material.rst

```diff
@@ -116,6 +116,11 @@ Disable Ambient Light
 Makes the object not receive any kind of ambient lighting that would
 otherwise light it.
 
+Ensure Correct Normals
+~~~~~~~~~~~~~~~~~~~~~~
+
+Fixes normals when non uniform scaling is used.
```

Comment:
Should be "non-uniform" or "nonuniform" (both are acceptable, although the latter is more common).